src/app/tests/integration/chip_im_responder.cpp

```diff
@@ -208,6 +208,7 @@ int main(int argc, char * argv[])
 
     chip::app::InteractionModelEngine::GetInstance()->Shutdown();
 
+    chip::DeviceLayer::PlatformMgr().StopEventLoopTask();
```

Comment:
This does not make sense to me.  This is after `RunEventLoop` has returned, so what does it even mean to call this function here?

Reply:
Acked, we don't need to explicitly stop event loop after RunEventLoop returned